paasta_tools/kubernetes_tools.py

```diff
@@ -1576,6 +1580,10 @@ def get_pod_template_spec(
             for registration in self.get_registrations():
                 labels[f"paasta.yelp.com/registrations/{registration}"] = "true"  # type: ignore
 
+        # Add istio sidecar injection label
```

Comment:
this comment doesn't add much, so either expand (explain what uses the label) or delete it.